cloudbuild.yaml

```diff
@@ -1,6 +1,6 @@
 steps:
```

Comment:
Can you please add:
```
options:
  defaultLogsBucketBehavior: REGIONAL_USER_OWNED_BUCKET
```
to the bottom of the cloud build file. This will fix the unit tests running in GCP